### DISCUSSION TITLE
Why does it keep rejecting my app for Github student developer program?

### DISCUSSION THREAD

**Discussion by arnav-sinha1** [community/community#59732] [GitHub Education]
### Select Topic Area

Question

### Body

My id proof is clear. All the information is double checked by me. My summer holidays are going to end. it is going to be a serious problem if I don't get it before my summer holidays end as I have exams. If any employee from GitHub is seeing this, please contact me, I will provide all info personally but please get this done before my summer holidays.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @arnav-sinha1 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by infinitelarge** [comment]
Can you provide more on why you are rejected
